.github/workflows/ci.yml

```diff
@@ -69,7 +69,7 @@ jobs:
         env:
           POSTGRES_HOST_AUTH_METHOD: trust  # never do this in production!
         # Set health checks to wait until postgres has started
-        options: --health-cmd pg_isready --health-interval 10s --health-timeout 5s --health-retries 5
+        options: --health-cmd pg_isready --health-interval 10s --health-timeout 5s --health-retries 5 -U postgres -d postgres
```

Comment:
```suggestion
        options: --health-cmd pg_isready --health-interval 10s --health-timeout 5s --health-retries 5 --username postgres --dbname postgres
```

Reply:
Also might be an args order problem - wrap all of `health-cmd` args in a single quoted string